feat(changelog): domains-and-dns-changed-update-pricing-for-art-at-de-it 2024-06-10 (#3282)
scaleway/docs-content: changelog/june2024/2024-06-10-domains-and-dns-changed-update-pricing-for-art-at-de-it.mdx

---
title: Update pricing for art, at, de, it, lu, nl, pl, tm,, ws
status: changed
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #domains channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2024-06-10
category: network
product: domains-and-dns
---

> Some registers are increasing in a significative way their pricing for their extensions. As a registrar, we have to follow this increase. Here are the changes : 
> - .art (Creation, Renewal, Transfer) : 26,23€HT
> - .at  (Creation, Renewal, Transfer) : 20,41€HT
> - .it (Creation, Renewal) : 7,81€HT
> - .lu (Creation, Renewal) : 27,78€HT
> - .nl (Create, Renewal) : 7,71€HT
> - . pl (Creation, Renewal, Transfer) : 34,98€HT
> - .tm (Creation, Renewal, Transfer) : 558,94€
> - .ws (Renewal) : 30,58€HT

Reviewer 2: ```suggestion
Due to significant price increases by certain registries, we are adjusting our prices accordingly. The new pricing for domain creation, renewal, and transfer are as follows:

- **.art** (Creation, Renewal, Transfer): 26.23€ HT
- **.at** (Creation, Renewal, Transfer): 20.41€ HT
- **.it** (Creation, Renewal): 7.81€ HT
- **.lu** (Creation, Renewal): 27.78€ HT
- **.nl** (Creation, Renewal): 7.71€ HT
- **.pl** (Creation, Renewal, Transfer): 34.98€ HT
- **.tm** (Creation, Renewal, Transfer): 558.94€ 
- **.ws** (Renewal): 30.58€ HT
```

---
title: Price adjustments for domain extensions
status: changed
author:
    fullname: 'Join the #domains channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-06-10
category: network
product: domains-and-dns
---

> Due to significant price increases by certain registries, we are adjusting our prices accordingly. The new pricing for domain creation, renewal, and transfer are as follows:

Reviewer: ```suggestion
Due to a significant price increase by certain registries, we are adjusting our prices accordingly. The new pricing for domain creation, renewal, and transfer is as follows:
```
